nixos/modules/services/backup/borgbackup.nix

```diff
@@ -581,7 +591,7 @@ in {
             default = "";
             example = ''
               # To add excluded paths at runtime
-              extraCreateArgs="$extraCreateArgs --exclude /some/path"
```

Comment:
as you had to change the example, it means that it's a breaking change worthy of a mentions in release notes, probably

Reply:
Agreed.